src/applications/find-va-forms/containers/SearchResults.jsx

```diff
@@ -79,11 +84,32 @@ class SearchResults extends Component {
     );
   };
 
+  onPageSelect = page => {
+    const { results, updatePagination } = this.props;
+
+    // Derive the new start index.
+    let startIndex = page * MAX_PAGE_LIST_LENGTH - MAX_PAGE_LIST_LENGTH;
+
+    // Ensure the start index is not greater than the total amount of results.
+    if (startIndex >= results.length) {
+      startIndex = results.length - 1;
+    }
+
+    // Update the page and the new start index.
+    updatePagination(page, startIndex);
+  };
+
   sortResults = (fieldLabel, selectedFieldOrder = ASCENDING) => {
-    const { results, updateResults } = this.props;
+    const { results, updatePagination, updateResults } = this.props;
 
     // Update local state for SortableTable.
-    this.setState({ selectedFieldLabel: fieldLabel, selectedFieldOrder });
+    this.setState({
+      selectedFieldLabel: fieldLabel,
+      selectedFieldOrder,
+    });
+
+    // Reset pagination values.
+    updatePagination();
```

Comment:
Can we add a unit test for this function, and for the `Pagination` component during render?